katello/katello-restore

```diff
@@ -91,7 +92,11 @@ def confirm
   print "Are you sure(Y/N)? "
   response = gets.chomp
   if /[Y]/i.match(response)
+    @time = Time.now
+    puts "Starting restore from #{DIR}: #{@time}"
```

Comment:
nitpick: consolidate code with `puts "Starting restore from #{DIR}: #{Time.now}"` and remove previous line